claat/render/md.go

```diff
@@ -262,3 +262,10 @@ func (mw *mdWriter) header(n *types.HeaderNode) {
 		mw.writeBytes(newLine)
 	}
 }
+
+func (mw *mdWriter) youtube(n *types.YouTubeNode) {
+	mw.newBlock()
+	mw.writeString(`<video id="`)
```

Comment:
Could this be one writeString call using Sprintf instead?

Reply:
Good call, done